src/core.py

```diff
@@ -132,20 +131,15 @@ def _move_modules(self, temp_repo, destination):
         :param string temp_repo: the folder containing the code.
         :param string destination: the folder where the add-on should end up at.
         """
-        folders = self._get_module_folders(temp_repo)
-        for folder in folders:
+        for folder in self._iter_module_folders(temp_repo):
             force_move(folder, destination)
 
-    def _get_module_folders(self, temp_repo):
-        """Get a list of module paths contained in a temp directory.
+    def _iter_module_folders(self, temp_repo):
+        """Iter over a list of module paths contained in the git directory.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/gh/Numigi/gitoo/pullRequest?prid=6195482)